### ISSUE TITLE
Passive verb et-obj marker

### ISSUE THREAD

**Issue by IsraelLand** [IAHLT/UD_Hebrew#63]
@amir-zeldes

> תנאי להעסקתו הוא כי [הוא] **ייבדק** תחילה **בדיקה רפואית** - [he] will be examined an examination

Similar to the (in?)famous [issue](https://github.com/UniversalDependencies/docs/issues/832), this seems to be an "internal object" of some sort. What's more fun is that it's passive, similar to - 

> הגד הגד לי כל אשׁר עשׂית 

Due to reasons of sanity to all involved parties, I would ask this here (but could be linked if need be) - what would you prefer here?
It seems obj for passive is a big no-no, but there you have it, the et object marker.
"Et" doesn't appear in this particular indefinite context of course, but the definite would contain it - 

> כי ייבדק תחילה את הבדיקה הרפואית - This seems regular **to me**
> כי ייבדק תחילה הבדיקה הרפואית? - this seems highly weird **to me**
>It may be that both are weird?

So apparently we can -
1. Obj, despite. This seems most natural to me, but you know.
2. Obl(:npmod?), if we assume the rephrasing `הוא יבדק על ידם (ב)בדיקה רפואית` but this is taking much liberties, imo
3. Your dislocated idea, which I'm not sure fits here, for only one NP (hu is implicit)
4. Give up on Hebrew altogether

Thank you

**Comment by amir-zeldes** [comment]
Sorry for taking longer to answer this, it got pushed down by a bunch of other messages... In the infamous issue I suggested subtyping obj, but TBH I also dislike introducing deprels that are extremely rare. So maybe 2., npmod is the least offensive idea of all. Have you polled the others? (maybe removing option 4 ;)

**Comment by IsraelLand** [comment]
Thanks.
Not officially polled, but I think nobody likes option 2 but me. יבדק את הבדיקה הרפואית works for you? 
If so, I can ask about the subtype (even though, I guess nobody really wants to go this route).
I'm not sure about the dislocated thingy.
So I guess obl (if the obj:sub is possible, I'd like to get your take on it even if we do go obl...)?

Regarding this, is the pov that when some other NP "takes" the slot, we revert to obl, correct?
So for example, two objects are not allowed\frowned upon in UD. And when this happens, we tend to solve this by obl - Netanel had the example of *הוא שכנע את משה את זה  - which we would revert to הוא שכנע את משה **ב**זה 
I'm not sure I agree this is ungrammatical per se, but I totally see the logic and this may be the logic behind the implicit ב of בדיקה רפואית.

His question specifically was about

הפקיד **השתכנע** כי ההורה השני אינו מקיים קשר עם ילדיו

CCOMP? It's only for OBJ by strict definition. a clausal obj is difficult for a mid verb.
Alternatively, CSUBJ? the NSUBJ slot is already taken.
That's how **I** understood it, at least. Both aren't mutually exclusive, really, but seem to have an effect one on the other.

So, 
1. ccomp\csubj for the sentence above?
2. this is a shot in the dark, but if this is a special kind of cobl not fitting other tags, perhaps advcl?
3. Is the general notion regarding "taken spots" affecting taggings, correct? Both for my sentence and Netanel's.

I hope I adequately conveyed why I see both as having something to do with the other.
Other similar cases - 
a. "sim lev ki..." - pay attention that - depreled **ccomp** even though the obj spot is already "taken" by attention\lev? 
b. "hu sha'al oto et hashe'ela" - intuitively, double obj (supposed iobj) which should be avoided? obl is weird, both are obj

Thank you

**Comment by amir-zeldes** [comment]
> obj:sub

I think that's probably out...

> Regarding this, is the pov that when some other NP "takes" the slot, we revert to obl, correct?

I think that's Dan and Joakim's view anyway, they feel very strongly about not having 2 objs. There's also the issue that, even if you think this is 'right' in some case, preventing it categorically by the validator rescues the big UD corpus from many more errors than the good that this would do.

> הפקיד השתכנע כי ההורה השני אינו מקיים קשר עם ילדיו

I think this is similar to English recipient passivization, and then the pakid is subject (here not passive, due to hitpael), and yes, the clause stays `ccomp`. Probably the Prague view would be to make the recipient in an active sentence (the person being convinced) an `iobj`, even if it appears with 'et', since when there are two obj, they will always demote one to `iobj`. For cognate objects I think maybe obl:npmod makes more sense than iobj.

> "sim lev ki..." - pay attention that - depreled ccomp even though the obj spot is already "taken" by attention\lev?

I think the validator doesn't prohibit it, so a lot of TBs have this... but yeah, it's questionable based on the single object slot policy.

> "hu sha'al oto et hashe'ela" 

This is closer to the internal or cognate object construction, although it's odd since the 'question' is a referring expression. I think the choices are 'obl:npmod' for question (or whatever we choose for cognate object), or `iobj` for the person (this is what UD English does)

**Comment by IsraelLand** [comment]
Thank you!
So to sum up ahead of Thursday (which overlaps with questions here, certainly with "hapakid hishtachnea" & "sim lev")-


"true" double objects - "hu sha'al oto et hashe'ela" - either **obl:npmod**, or introducing **iobj** 
(I think Hebrew is quite similar to the [Czech here](https://universaldependencies.org/u/dep/iobj.html))
(With the added complication of passive cases like "hu yibadek bdika refuit" to keep in mind when deciding)

Obj positions that are "taken" by expression - "sim lev ki" - realistically, still ccomp, but has some theoretical limitations (perhaps cobl-advcl?)

mid\pass verbs with ccomp -  "hapakid hishtachnea" - realistically, still ccomp, even though not the classic obj-position for ccomp, unless we'd like to keep the csubj-ccomp scale more rigid

Thanks

**Comment by amir-zeldes** [comment]
Just noticed I forgot to answer in this issue too: yes, we should develop a clear policy on this, and I think obl:npmod for cognate object and iobj for double object is the most feasible. I can't remember in which issue this was asked, but about the `Case`, yes, it would still be `Acc` even for the `iobj` item. 

For "sam lev ki" I would say ccomp is the same as obj, so either we treat the clause as advcl, or decide that idioms like "sam lev" get a non-obj analysis, e.g. compound (in some languages we see `compound:lvc` for light verbs), then we can keep ccomp here.

"hapakid hishtachnea ki" is ccomp, because if there is only one object you can keep it (though some corpora, like GUM, distinguish iobj from obj even if only one is used), and either way, even if you want to distinguish indirect recipient arguments, this isn't one - I think it would be "hapakid" which would be iobj if it were active:

Dani shixnea et hapakid/iobj she-yelex/ccomp

**Comment by IsraelLand** [comment]
Thank you.
As a matter of fact I just finalized that batch - "sam lev ki" so I'd settle for advcl.

"hapakid histachnea" -
הפקיד שוכנע כי יש להביא בחשבון תוספות שכר אלו
same goes for passive? So long as there is one, not two "taken positions" competing for the same position, ccomp? (same goes for "ratz ritza" vs. "shaal oto et hasheela" - same object, but no iobj if only one place. It's still quite peculiar cause they're the same, ratz et haritza & shaal et hasheela oto so that's awkward to say one is npmod.)

Thank you

**Comment by amir-zeldes** [comment]
It depends on whether you adopt an EWT-like policy (if one unmediated non-subj arg, then it is obj/ccomp) or a GUM-like one (args get labeled based on the valency slot which they saturate). GUM has cases of iobj without obj, EWT does not.

**Comment by IsraelLand** [comment]
Yeah I see. I think looking at valency for a language with very few true iobj cases, is a bit of an overkill. 
So, only iobj for cases where we already have obj (would that mean "ratz ritza" is equated to "shaal sheela" - both obj, but when "shaal oto sheela" iobj is introduced?). What would you prefer?
Thank you

**Comment by NathanD38** [comment]
> It depends on whether you adopt an EWT-like policy (if one unmediated non-subj arg, then it is obj/ccomp) or a GUM-like one (args get labeled based on the valency slot which they saturate). GUM has cases of iobj without obj, EWT does not.

Dani/nsubj shixnea et ha-pakid/iobj she-hu higish et kol ha-mismaxim/ccomp

Ha-pakid hishtaxnea ki Dani higish et kol ha-mismaxim

Ha-pakid shuxna ki Dani higish et kol ha-mismaxim.

Evidently, in the middle voice, pakid is in the nsubj position syntactically, but in the iobj saturated slot in terms of valency operations. That would cause the sentence to become ungrammatical, at least in 3rd person sg/pl, if we were to give
pakid iobj. 

*hishtaxnea ki Dani higish et kol ha-mismaxim.

1st and 2nd person are grammatical sans-subject in Hebrew (in the Past tense):

hishtaxnati/hishtaxnata ki Dani higish et kol ha-mismaxim.

I don't think this is possible in English or other languages in which a subject position must be filled, and there are no
such pronominal suffixes to indicate the subject:

*[I/You/She] was convinced that Dani handed all the (required) forms.

But what is the valency slot that the clause saturates in the middle and passive examples, השתכנע and שוכנע?
Does it change in any way? It can be replaced with an oblique phrase containing a demonstrative prnoun in both,
seemingly with no different of meaning, and that is true for the active voice as well:

Dani/nsubj shixnea et ha-pakid/iobj be-xax.

Ha-pakid hishtaxnea be-xax. 

Ha-pakid shuxna be-xax. 

If this cannot be ccomp by GUM's standards, what should the clause be?

Thank you!

**Comment by amir-zeldes** [comment]
> What would you prefer?

It looks like if there's a proper cognate object we can use obl:npmod, but for things like shixnea et hapakid sheyelex it's necessary to do iobj + ccomp

> Evidently, in the middle voice, pakid is in the nsubj position syntactically, but in the iobj saturated slot in terms of valency operations. That would cause the sentence to become ungrammatical, at least in 3rd person sg/pl, if we were to give
pakid iobj.

Actually passivization of iobj is not so unusual an English allows it as well:

  * Kim gave me the documents
  * The documents were given by Kim
  * I was given the documents

So I think this is not an argument against "Dani shixnea et ha-pakid/iobj she-yelex/ccomp"

> I don't think this is possible in English

I mean, Hebrew is pro-drop, so I take "hishtaxnati" to be equivalent to "I became convinced" or similar. Basically there is a subject in both cases, just not as a token in UD.

> Dani/nsubj shixnea et ha-pakid/iobj be-xax. Ha-pakid hishtaxnea be-xax. Ha-pakid shuxna be-xax.

Only the first of those is problematic (example 2 is nsubj + obl, 3 is nsubj:pass + obl). If you take the GUM policy, you would get:

  * Dani/nsubj shixnea et ha-pakid/iobj be-xax/obl

EWT policy would be:

  * Dani/nsubj shixnea et ha-pakid/obj be-xax/obl

Since there are no longer two objects, so there can be no iobj. GUM examples with iobj but no obj:

http://universal.grew.fr/?custom=627530de43307

Some examples of "tell" in EWT without iobj or ccomp but a single obj:

http://universal.grew.fr/?custom=6275313955680

Not all of these are the construction in question, precisely because in EWT it's impossible to tell whether there is only the theme or only the recipient.

**Comment by IsraelLand** [comment]
So if I got it right,
1. iobj+ccomp for stuff like **shixnea et hapakid sheyelex**  where both obj-like-positions are saturated 
(but "regular" obj if no ccomp, as well as ccomp if no other object)
2. npmod for cognate objects like **shaal oto et hasheela** (Even though I don't see why cognate objects are to be treated differently than "regular objects", i.e. obj+iobj instead of obj+npmod, nor do I have a clear way to differentiate the two)
3. and I assume we cannot find a "true" iobj+obj case in Hebrew, barring cognate objects (npmod) or clausal ones (ccomp)?

Let me rephrase Netanel's questions and leave the obl's to the side, are these - 

 4. Ha-pakid hishtaxnea ki Dani higish et kol ha-mismaxim - ccomp (as previously mentioned)?

5. Ha-pakid shuxna ki Dani higish et kol ha-mismaxim - ccomp as well? (nsubj is saturated, so no double-csubj I assume).

**That is if we go by EWT**.

7. I assume iobj+ccomp **shixnea et hapakid sheyelex** is still abiding by EWT (assuming ccomp is obj-like in this case)?

8. I'm asking because I look at this as absolute, like you said, if we start going the GUM way we need to always mark by valency for (i)objects, which we wouldn't want to do 
 (**I might be wrong and we can apply GUM valency tagging only for the very specific ccomp+iobj, while otherwise remaining EWT**)

Thank you!

**Comment by amir-zeldes** [comment]
1. Yes
2. I admit this is tricky, and I'm not sure "shaal oto et hasheela" is even a cognate object in the traditional sense of 'musa pnimi', since it is definite (we are possibly referring to a concrete question, which we have already talked about, so it's an entity term). At the other end of the spectrum, we have Biblical "sha'ol sha'alti" (which can only be a cognate object, and is not back-referenceable either), and indefinite cases in between, such as "ratsiti lish'ol otxa she'ela" (which is arguably borderline). 
3. I think it's possible to get iobj equivalents of some clauses, esp. infinitives: 
  * limadeti oto/i?obj lashir/xcomp
  * limadeti oto/iobj et hashir/obj
4. yes
5. yes, ccomp

I'm not sure what happened to 6. :)

7. Yes, in theory, though I think in practice EWT sometimes doesn't follow through on this. But based on the guidelines, yes.
8. I agree that iobj is rarer in Hebrew, so it's less motivated to go the GUM way. 

We had mainly had it that way in GUM since before UD (from Stanford Dependencies), and felt bad about throwing away the difference between "tell a story" and "tell people" - it's trivial to collapse them if we want to. Then it turned out some other corpora were like GUM (esp. some Germanic and Slavic, due to having explicit dative marking), and we decided not to revise it in the hope the guideline will change later (in some cases, such as nested copula and parataxis for indirect speech, this actually happened, so we were happy we never implemented those changes)

**Comment by IsraelLand** [comment]
Thank you.

2.  Yeah I see, tying in with 3, 
3.  it's a good example. You think changing the order of objects can be a test for cognate objects\iobjs?

*hu heexil et hadaisa et hatinok
*hu shaal et hasheela oto
?limadeti et hashir oto

The first two are extremely awkward, not that much for the third, at least to my intuition. Otherwise I'm not sure what guideline could be made to differentiate between npmod+obj for cognates vs. iobj+obj for 2 non-cognate objects.

Right, so to summarize (mostly to myself, later to the guidelines),
a. we introduce iobj+obj for cases with **double** object positions, but not for cognate objects, which get npmod(+obj)
b. ccomps in general don't have to be in the object position (shuxna ki, hishtaxnea ki are still ccomp)
c. they can be though, so in that case (shixnea oto ki) we utilize iobj+ccomp, since that ccomp saturates the obj position
(for idioms like sam lev ki, the direct obj is saturated so the clause (ki...) gets tagged as advcl)


EDIT: Netanel pointed out the xcomp example. Would infinitival xcomp "limed oto lashir" warrant iobj instead of obj? Would **any** xcomp (with another object in the sentence) warrant such treatment? 
I'm asking because I feel xcomp is quite more inclusive than mostly-objectival ccomp, I might be wrong, but in any case I think it's much more prevalent than the rare "limadeti oto et hashir"\"limadeti oto shehashir tov".

Thank you

**Comment by amir-zeldes** [comment]
> You think changing the order of objects can be a test for cognate objects\iobjs?

I think all of the reorderings are awkward, but none of them are totally ungrammatical, since Hebrew word order is essentially free as long as phrases stay together (compared to English, where such reorderings generally sound ungrammatical, in Hebrew I only feel they are unusual, not totally wrong). I suppose the best test is to ask whether this argument is 'normal' for the verb, in addition to being a cognate. For example, I think the following distinguishes rats ritsa and shaal sheela:

  * shaalti mashehu 
  * *ratsti mashehu

So, shaalti is transitive, and essentially licenses that 'question' object even without cognate status. Ratsti doesn't. And I don't just mean this works for intransitives, consider transitive::

  * katavti mixtav
  * katavti mixtav ktiva tama (internal obj. IMO, obl:npmod)
  * katavti mashehu (can only be interpreted as saturating the same object as in the first example, not the second)
  * *katavti mixtav mashehu (intended interpretation: "I wrote a letter some specific way")

> Would infinitival xcomp "limed oto lashir" warrant iobj instead of obj

No, it's normal for xcomp to appear with obj, that's the canonical UD treatment of causatives and secondary predication on transitives.

**Comment by IsraelLand** [comment]
Right. The first one works well, but indeed it doesn't do much for heexalti et hatinok et hadaisa, and so for transitives, you meant
katavti [mixtav ktiva tama] or [mixtav] [ktiva tama]
not as a compound, but as another element?

ktiva tama is obl:npmod, sure, but is it the same npmod as an internal object :npmod?
I have to say, for my intuition, at least, I only got your meaning when definite, meaning 
katavti mixtav ktiva tama - is always "I wrote a fine-handwriting letter"
while
katavti et hamixtav ktiva tama - would mostly be "I wrote the letter [in] a fine handwriting"

so I don't think 
*katavti (et ha)mixtav et haktiva hatama
works at all like 
heexalti (et ha)tinok et hadaisa
which can be indef\def quite freely (except heexalti et hadaisa et hatinok imo),
while the first one doesn't, because it seems to be a different npmod

so your fourth example, is similar to the second imo, both impossible (at least when indef.)

Thank you

**Comment by amir-zeldes** [comment]
Sorry I wasn't clear: yes, I meant two phrases, "I wrote the letter, (in) fine writing". This is traditionally considered a type of internal object, where the purpose of the construction is to exploit the cognate object for hanging an ADJ on it, resulting semantically in an adverbial modification of the verb. In traditional Arabic grammar, bare cognate objects are known as [maf'uul mutlaq](https://www.google.com/books/edition/Arabic_Grammar_and_Linguistics/_TMsBgAAQBAJ?hl=en&gbpv=1&dq=maful+mutlaq&pg=PA110&printsec=frontcover), and the type with adjective is called naa'ib maf'uul mutlaq, which is pretty common. 

I think the possibility of a definite article is not an absolute test, but I agree that the classic 'inner object' in traditional grammar is usually indefinite (like rats ritsa). The distinguishing point I was trying to make is whether or not this argument is really licensed in any special way by the verb's valency (a true object), since cognate objects can be added to *anything*, meaning they are not really objects (for example "afiti et halexem afiya yesodit" -- the 'baking' is just the event itself, not a special object of 'bake', a slot which is occupied by the bread in this example).

**Comment by IsraelLand** [comment]
I see. Both npmod's are just different manifestations of the same cognate object phenomenon, one adverbial.
1.
What about "heexalti..." then? 
"tinok" is definitely a true object, do you think "daisa" to be so as well? They do work when separate.
How should we distinguish them, then? Should we employ obj,iobj?
(One could argue "heexalti tinok [be\et-ha]maaxal taim", where "maaxal taim" is standalone and interchangeable w. 
"daisa", but I'm not sure this to be the case)

2. ccomp for passive "shuxna ki ein tzorex" - should we introduce ccomp:pass (consistency along the chain) or allow ccomp for passive (validation wise)?

Thank you

**Comment by amir-zeldes** [comment]
> What about "heexalti..." then?

Yes, they are 'true' objects, integrally connected to the argument structure of that verb, so tinok/iobj daisa/obj. Notice that you can hypothetically still add the cognate object:

  * heexalti et hatinok et hadaisa haaxala mehira

> ccomp for passive

No, there is no `ccomp:pass` - it's not unusual for ditransitive verbs to allow passivization with one object as the new subject (nsubj:pass) and the other remaining an obj or ccomp. The same happens in English for:

John/nsubj:pass was sent letters/obj

**Comment by IsraelLand** [comment]
I see. Thank you very much!

**Comment by NathanD38** [comment]
@amir-zeldes 

In the spirit of cognate objects, are the following examples considered `obl:npmod` or regular `obj`?

שינויים דמוגרפיים אלה עלולים לגרום לקשיים במימון הגידול שייווצר בהיקף הקצבאות השונות
_ולהטיל_ **נטל כבד** יותר על מערכות הביטחון הסוציאלי,

הסדר פרישת החובה _פוגע_ **פגיעה קשה** בזכותו החוקתית של העובד לשוויון ואף בחופש העיסוק, 
וזאת באופן העולה על הנדרש.

**Comment by amir-zeldes** [comment]
I would say the first is obj (it's a collocation, but that doesn't mean "lehatil" is any less a transitive verb), whereas the second is a cognate obj, and therefore npmod (notice that "pogea oto" is bad, since it is not actually a transitive verb, except for the possibility of adding the cognate obj, which all verbs have in Hebrew).

**Comment by NathanD38** [comment]
@amir-zeldes 
Thank you!

**Comment by IsraelLand** [comment]
What about - 

פורסמה חבילה ש**תעלה 20 ש"ח** לחודש לאחר הסבסוד
**מחייגים 198** מכל טלפון

I feel an obl:npmodness about them, or obj, WDYT? (I assume no nsubj, because **[hi] taale**, **[atem] mexaigim**)
Thanks

**Comment by amir-zeldes** [comment]
For the second one I feel definitely obj (you dial a number, "et hamispar"). For the first it's a bit less obvious, but the English corpora use `obj` with "costs (money)" so I guess obj for both.

**Comment by IsraelLand** [comment]
Thank you!
